# 現在のモードを取得する(SCIBOT-Xのみ)
---

